targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H747xI/TARGET_PORTENTA_H7/COMPONENT_WHD/generated_mac_address.txt

```diff
@@ -0,0 +1,9 @@
+/* 
+ * This file is used to set the MAC address in NVRAM.
+ * The MAC address of the Wi-Fi device may be configured in OTP and/or in NVRAM. 
+ * If both OTP and NVRAM contains the MAC address then OTP programmed MAC address will be used. 
+ * PSOC boards are usually programmed with OTP MAC address. 
+ * MAC address is printed during WHD power up
+ */
+
+#define NVRAM_GENERATED_MAC_ADDRESS        "macaddr=00:A0:50:6f:b2:ea"
```

Comment:
Should this macro be define in some json file in order to be reconfigurable ?

Reply:
That's just a placeholder (the real MAC address is written in the silicon) but that's the method Cypress uses so I kept is